docs/source/Scripts/Common/common.rst

```diff
@@ -2,10 +2,16 @@
 Common
 ======
 
+.. sidebar:: Script under external
+
+    The external directory is a subdirectory of common.
+    All scripts in there come under common.
+    
```

Comment:
`.. sidebar:: External Contents`

"The External directory contains wrappers respective to each utilized external program"